### PR TITLE
openfst 1.6.3

### PR DESCRIPTION
**Pull request by stevenbedrick**
### Have you:

- [X] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [X] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [X] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [X] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [ ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [X] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [X] Checked that the tests still pass?

### CODE REVIEW

**Comment by stevenbedrick** [comment]
The CI test failed because, apparently, zlib wasn't available on the include path in the container. I've never seen this problem before- is zlib really not on there?

File: openfst.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 class Openfst < Formula
   desc "Open-source library for working with weighted finite-state transducers."
   homepage "http://www.openfst.org/"
-  url "http://openfst.org/twiki/pub/FST/FstDownload/openfst-1.6.1.tar.gz"
-  sha256 "5245af8ebccb96208eec2dfe3b3a81143d3565a4d41220bff299287fb3333f7d"
+  url "http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-1.6.3.tar.gz"
+  sha256 "5c28b6ccd017fc6ff94ebd0c73ed8ab37d48f563dab1c603856fb05bc9333d99"
+
+  revision 1
```
Comment (iMichka) [inline openfst.rb]: Please don't add a revision line here. The version bump due to the URL change is enough

(The mac bot also tells it):
> homebrew/science/openfst:
>  * 'revision 1' should be removed

Comment (stevenbedrick) [inline openfst.rb]: Oh, ok, thanks for pointing that out!

**Comment by stevenbedrick** [comment]
Gah, forgot to bump the revision line in the two libraries that depend on openfst, so they're failing. I seem to forget this every single time...

**Comment by stevenbedrick** [comment]
Hello! Looks like all the CI tests pass, and I've made the other requested change; is this ready to merge?

**Comment by iMichka** [comment]
@stevenbedrick Thanks for your contribution. I just made some history rewriting before merging; so that we follow our convention for the git history. I squashed your 2 first commits into one, and split the revision bumps, and changed the title accordingly. This is just a small comment for your next revision bumps :)

**Comment by stevenbedrick** [comment]
Thanks! I will try and organize things better next time. :-)

-SB (mobile)

**Pull request closed without merging**